scripts/skills/perks/perk_rf_discovered_talent.nut

```diff
@@ -27,13 +27,17 @@ this.perk_rf_discovered_talent <- ::inherit("scripts/skills/skill", {
 		if (potential.len() == 0)
 			return;
 
+		::Math.seedRandom(actor.getUID() + ::toHash(this.getID()) + this.m.AttributesRolled.len());
+
 		local choice = ::MSU.Array.rand(potential);
 		local toAdd = ::MSU.Class.WeightedContainer([
 			[70, 1],
 			[20, 2],
 			[10, 3]
 		]).roll();
 
+		::Math.seedRandom(::Time.getRealTime());
```

Comment:
Since `resetting the seed` seems to be the exception, you could add an explanation, why this is needed here